tuf/ngclient/config.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+"""Configuration options for Updater class
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass
+class UpdaterConfig:
+    max_root_rotations: int = 32
+    max_delegations: int = 32
+    default_root_max_length: int = 512000  # bytes
+    default_timestamp_max_length: int = 16384  # bytes
+    default_snapshot_max_length: int = 2000000  # bytes
+    default_targets_max_length: int = 5000000  # bytes
```

Comment:
None one of the variables should be called "default" i think -- looks a bit odd in the code since they might be set by the user and they're still called "default"?

Reply:
Maybe default makes sense for snapshot and targets where this is the value used if the one in metadata is not se but I renamed all anyway 520b344.